### PR TITLE
fix(frontend): modify LogoImage test to see if it fixes race condition bug

### PR DESCRIPTION
**Pull request by Bettelstab**
## 🍰 Pullrequest
We still have the VImg test error sometimes, which is annoying and slows down development because CI blocks and you are never sure if you just introduced a new error.

This is an attempt to fix it. I examined all test files where VImg is used without an VApp context, and this file is the only one I found.

If it doesn't fix the problem, it won't hurt either.

### CODE REVIEW

File: frontend/src/components/menu/LogoImage.test.ts [hunk 1]
```diff
@@ -32,10 +38,8 @@ describe('LogoImage', () => {
     })
 
     describe('size large', () => {
-      beforeEach(async () => {
-        await wrapper.setProps({
-          size: 'large',
-        })
+      beforeEach(() => {
+        wrapper = Wrapper({ size: 'large' })
```
Comment (roschaefer) [inline frontend/src/components/menu/LogoImage.test.ts:42]: Is there a missing

```suggestion
        wrapper = await Wrapper({ size: 'large' })
```

maybe?

File: frontend/src/components/menu/LogoImage.test.ts [hunk 2]
```diff
@@ -19,7 +25,7 @@ describe('LogoImage', () => {
   })
 
   it('renders Logo', () => {
-    expect(wrapper.element).toMatchSnapshot()
+    expect(wrapper.findComponent(LogoImage).element).toMatchSnapshot()
```
Comment (roschaefer) [inline frontend/src/components/menu/LogoImage.test.ts:28]: Weird, the snapshots haven't been updated?

**Review by roschaefer — APPROVED** [review]
There is nothing blocking this PR from my side.

let's try

File: frontend/src/components/menu/LogoImage.test.ts (hunk 2 above)
Comment (Bettelstab) [reply frontend/src/components/menu/LogoImage.test.ts:28]: Exactly, as I just want to test what the component renders, and that is still the same.

File: frontend/src/components/menu/LogoImage.test.ts (hunk 1 above)
Comment (Bettelstab) [reply frontend/src/components/menu/LogoImage.test.ts:42]: Don't think so, mount is not async.

**Pull request merged**